config/locales/en.yml

```diff
@@ -47,8 +47,8 @@ en:
             style_in_template: "<style> tag in %{template}. Use an app.css file instead."
             no_code_for_ifo_notemplate: Javascripts, stylesheets, and templates are
               not allowed when an iframe URI or noTemplate is specified
-            invalid_default_locale: "%{defaultLocale} is not a valid default locale."
-            missing_translation_file: 'Missing translation file for locale ''%{defaultLocale}''.
+            invalid_default_locale: "%{default_locale} is not a valid default locale."
```

Comment:
In theory, this would need a new key. We will get validation errors, but once it is merged, I can update manually the placeholders for all current translations, is it ok @pdeuter ?